packages/modules/desktop-viewer-react/src/services/Initializer.ts

```diff
@@ -107,8 +107,12 @@ export class DesktopInitializer {
           DesktopInitializer._initializing = false;
           DesktopInitializer._cancel = undefined;
         });
-    } else {
+    } else if (this._initializing) {
       this._initialized = Promise.resolve();
```

Comment:
lets change this to something like

`if (!this._initialized) {
       this._initialized = Promise.resolve();
} 
return this._initialized;
`
and use the same logic in both the web and desktop Initializers